doc/lsst.ap.verify/datasets-creation.rst

```diff
@@ -26,15 +26,11 @@ This will create empty directories for all data and will add placeholder files f
 Organizing the Data
 -------------------
 
-.. TODO: talk to David/Meredith to confirm details of how this works, and how flexible the system is (DM-12851)
-
 * The :file:`raw` and :file:`calib` directories contain science and calibration data, respectively.
-  The directories must be organized in the same way as the observatory's normal data products, for compatibility with the appropriate ``obs`` package.
-* The :file:`templates` directory contains a :ref:`LSST Butler repository<butler>` containing processed images useable as templates. Template files must be ``TemplateCoadd`` files produced by a compatible version of the LSST science pipeline.
-
-.. TODO: are these more standardized? are they available from somewhere? (DM-12851)
-
-* The :file:`refcats` directory contains one or more tar'd HTM files containing astrometric and photometric reference catalogs.
+  The directories may have any internal structure.
```

Comment:
Please clarify whether this point applies only to `raw` and `calib` or to all the directories mentioned in this section.

Reply:
Is it confusing only because of the diff? This is how it looks in HTML:

* The `raw` and `calib` directories contain science and calibration data, respectively. The directories may have any internal organization.
* The `templates` directory contains a LSST Butler repository containing processed images useable as templates. Template files must be `TemplateCoadd` files produced by a compatible version of the LSST science pipeline.
* The `refcats` directory contains one or more tar files, each containing containing one or more astrometric or photometric reference catalogs in HTM shard format.

Reply:
Whoops, yes, the diff made it look like its own bullet point at a glance. All good!